tests/core/test_context.py

```diff
@@ -372,7 +448,7 @@ def test_override_builtin_audit_blocking_mode():
             call(
                 "Audit 'not_null' for model 'db.x' failed.\n"
                 "Got 1 results, expected 0.\n"
-                'SELECT * FROM (SELECT * FROM "sqlmesh__db"."db__x__2457047842" AS "db__x__2457047842") AS "_q_0" WHERE "c" IS NULL AND TRUE\n'
+                'SELECT * FROM (SELECT * FROM "sqlmesh__db"."db__x__1949820056" AS "db__x__1949820056") AS "_q_0" WHERE "c" IS NULL AND TRUE\n'
```

Comment:
Should we fix this test by actually interpolating the corresponding `snapshot.fingerprint.to_version()` instead of hard-coding it?

Reply:
sure I can change that, I thought these were left as reminders for breaking changes, to identify when the data hashes change and create a migration script